### PR TITLE
11 create project repository ah

### PR DESCRIPTION
**Pull request by AbiHaddon**
Amended structure 
added: requirements.txt, [installation.md](http://installation.md/), user_guide.md
moved files into the correct folder and added a note to the readme file

### CODE REVIEW

**Comment by AbiHaddon** [comment]
@Fr3dMer Just realised this branch is to merge into main .....

**Comment by Fr3dMer** [comment]
Hmm I think if you click edit you can change the target branch 
image

File: requirments.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# file that can be used to install a minimum basic set of Python modules / libraries using pip
+
+# The requirements.txt file has been either replaced by a pyproject.toml or integrated into a pyproject.toml
```
Comment (Fr3dMer) [inline requirments.txt:3]: #### SUGGESTION:
- It seems that pyproject.toml replaces the need for a setup.py, but does not provide a functional way to pin dependency versions like in Requirements.txt, for deployable applications it seems that both are required. This has taken me down a rabbit hole so let me know what you think 
- https://venthur.de/2022-12-18-python-packaging.html 
- https://github.com/pypa/packaging.python.org/issues/685

File: src/GenePanelRepo.db
Comment (Fr3dMer) [inline src/GenePanelRepo.db:1]: #### SUGGESTION:
- Best to have db removed from repo and have it generated during install?

File: .idea/.gitignore [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Default ignored files
+/shelf/
```
Comment (Fr3dMer) [inline .idea/.gitignore:2]: #### REQUEST:
- Needs to ignore python specific files 
- e.g: 
__pycache__/
*.py[cod]
*$py.class

File: .idea/inspectionProfiles/profiles_settings.xml
Comment (Fr3dMer) [inline .idea/inspectionProfiles/profiles_settings.xml:1]: #### COMMENT:
- Need to come back to this as I assume this is something from Petes tutorials

**Review by Fr3dMer — CHANGES_REQUESTED** [review]

File: requirments.txt (hunk 1 above)
Comment (Fr3dMer) [reply requirments.txt:3]: Interesting point thank you 😊

**Review by AmyGrimwood — APPROVED** [review]
Looks really good Abi.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+# A fully rendered markdown file (README.md) that describes the project
```
[the remark this answers is not in the document]
Comment (AbiHaddon) [reply README.md:1]: to be done next stage

File: docs/Installation.md [hunk 4]
```diff
@@ -0,0 +1,2 @@
+# An installation markdown file that tells us how to run the project
```
[the remark this answers is not in the document]
Comment (AbiHaddon) [reply docs/Installation.md:1]: to be done next stage

File: requirments.txt (hunk 1 above)
Comment (AbiHaddon) [reply requirments.txt:3]: to consider next stage

File: src/GenePanelRepo.db
Comment (AbiHaddon) [reply src/GenePanelRepo.db:1]: yes! will do

File: .idea/.gitignore (hunk 2 above)
Comment (AbiHaddon) [reply .idea/.gitignore:2]: to be done next stage

**Review by Fr3dMer — APPROVED** [review]
As per our conversation on teams, this was laying out the structure of repo. Further detail to be achieved in next sprint

**Pull request merged**